### ISSUE TITLE
[Feedback] Use semistandard everywhere

### ISSUE THREAD

**Issue by McDonnellDean** [nearform/nscale#80]
I was wondering what the opinion to using [semistandard](https://github.com/Flet/semistandard) in all nscale repos would be. It is the semi-colon version of [standard](https://github.com/feross/standard).

I personally think both of these will win out over other style checkers. If this is something ye find interesting I have no problem looking at doing a slow move to this one repo at a time?

@mcollina @pelger @Nss @darragh-hayes opinions?

**Comment by mcollina** [comment]
I agree, but it is really low priority. What's the others opinion on this?

**Comment by Nss** [comment]
Fine by me but I agree on the low priority.

**Comment by McDonnellDean** [comment]
Agree on the priority, its more just to get the green light on it.
